src/components/KTableView/KTableView.vue

```diff
@@ -333,6 +393,13 @@ const props = defineProps({
     type: Function as PropType<(row: Record<string, any>) => RowLink>,
     default: () => ({}),
   },
+  /**
+   * A function that conditionally specifies whether bulk actions are disabled for a row and the tooltip to display
```

Comment:
add a note about "Defaults to `true` as it requires that ... " that the other thing is set, etc.

Also add to the docs page?

Reply:
Done